### PR TITLE
Add `delete teamdrive` command

### PR DESCRIPTION
**Pull request by catthehacker**

### CODE REVIEW

**Comment by catthehacker** [comment]
need to find out how to get only files from root of the drive and delete them so it doesn't sit for hours deleting already deleted files

File: cmd/delete/teamdrive.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	}
+
+	driveApi, err := api.NewDriveService(tokenSource)
+	if err != nil {
+		logrus.Error(err)
+		return
+	}
+
+	if forceDelete {
+		var query string = "parents = \"" + teamdriveId + "\""
```
Comment (fionera) [inline cmd/delete/teamdrive.go]:
```
query := fmt.Sprintf(`parents = "%s"`, teamdriveId)
```

**Review by fionera — CHANGES_REQUESTED** [review]
Only this one string concatination :+1:

File: cmd/delete/teamdrive.go (hunk 1 above)
Comment (catthehacker) [reply cmd/delete/teamdrive.go]: Done

**Review by fionera — APPROVED** [review]

**Pull request merged**